src/libraries/black-listed-key-bindings.library.ts

```diff
@@ -28,10 +26,7 @@ export class BlackListedKeyboardShortcutChecker {
     private mappedBlackListedKeyBindings = this.blackListedKeyBindings.map((blkb) => mapKeyboardShortcutCombination(blkb));
 
     check(listener: IKeyboardShortcutListener): void {
-        // should only get checked in DevMode
-        if (!isDevMode()) {
```

Comment:
This is an important check.  I know why you removed it, since Jamil suggested doing so, but it has to be put back in.  I'll create an issue to put it back.